doc-General_Configuration/topics/RH_Access_Insights.adoc

```diff
@@ -1,31 +1,31 @@
 
 [[rh-access-insights]]
-== Red Hat Access Insights
+== Red Hat Insights
 
-*Red Hat Access Insights* is a new service that uses the collective knowledge to help end users proactively diagnose systems and avoid critical downtime situations. Red Hat Access Insights does this by having systems periodically check in similar to Red Hat Subscription Management.
+*Red Hat Insights* is a new service that uses the collective knowledge to help end users proactively diagnose systems and avoid critical downtime situations. Red Hat Insights does this by having systems periodically check in similar to Red Hat Subscription Management.
```

Comment:
@cbudz Do you think Red Had Insights is old enough now that we should remove the word "new"? Might also future-proof our docs a bit as well.